react/Components/Footers/FooterMobile.tsx

```diff
@@ -0,0 +1,200 @@
+// import { FaArrowRight } from "react-icons/fa";
+import style from "./FooterDesktop.css";
+
+interface FooterProps {}
+
+const FooterMobile: StorefrontFunctionComponent<FooterProps> = (props: any) => {
+  console.log(props, "------------");
+
+  const footerSectionItems = [
+    {
+      title: "Shop",
+      items: props.shopLinks ? props.shopLinks : [],
+    },
+    { title: "About", items: props.aboutLinks ? props.aboutLinks : [] },
+    { title: "Help", items: props.helpLinks ? props.helpLinks : [] },
+  ];
+
+  const [showList, setShowList] = useState(-1);
+
+  const handleClick = (index: number) => {
+    setShowList(index);
+  };
+  const handleClose = () => {
+    setShowList(-1);
+  };
+  return (
+    <div>
+      <div className="footer b-action-primary c-muted-1 pl5 pr5">
+        <div className="row  col-12 contain pa9-l pa0-s block">
+          {footerSectionItems &&
+            footerSectionItems.map((ele, index: number) => (
+              <div className="pt3">
+                <div className="flex justify-between">
+                  <b key={index} className="">
+                    {ele.title}
+                  </b>
+                  {showList == index ? (
+                    <button
+                      type="button"
+                      className="b--black-0125 bg-transparent"
+                      onClick={() => handleClose()}
+                    >
+                      <Icon id="nav-minus" handle="plusIcon" />
+                    </button>
+                  ) : (
+                    <button
+                      className="b--black-0125 bg-transparent"
+                      onClick={() => handleClick(index)}
+                    >
+                      <Icon id="nav-plus" handle="plusIcon" />
+                    </button>
+                  )}
+                </div>
+
+                {showList == index && (
+                  <ul className="pl0 ma0 " style={{ listStyle: "none" }}>
+                    {ele.items.map((menu: any, index: number) => (
+                      <React.Fragment key={index}>
+                        <li className="mt2">
+                          <a href={menu.url} className={`${styles.linkText}`}>
+                            {menu.text}
+                          </a>
+                        </li>
+                      </React.Fragment>
+                    ))}
+                  </ul>
+                )}
+              </div>
+            ))}
+        </div>
+      </div>
+      <div className="row  col-12 contain w-100 ph5">
+        <div className="flex flex-column">
+          <p className="b ">
+            Sign up to receive the project antelope newsletter
+          </p>
+          <input
+            type="email"
+            className={`${style.email}`}
+            placeholder="Enter your email address"
+          />
+          <span className={`${style.arrow}`}>
+            {" "}
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
```

Comment:
use vtex style guide for icons and please refer to the doc: [https://styleguide.vtex.com/#/Icons](https://github.com/akshay-trika/training-task/pull/1/url)

Ex: `import { Plus } from 'vtex.styleguide'`